tests/Makefile

```
@@ -44,32 +46,24 @@ env:
 
 
 build-plugins:
-	docker run \
-	    -u $(LIZMAP_USER_ID):$(LIZMAP_GROUP_ID) \
-	    --rm -i \
-	    -e QGSRV_SERVER_PLUGINPATH=/srv/plugins \
-	    -e QGIS_PLUGIN_MANAGER_SOURCES_FILE=/tmp/sources-plugin-manager.list \
-	    -e QGIS_PLUGIN_MANAGER_CACHE_DIR=/tmp/cache-plugin-manager \
-	    -v $(shell pwd)/qgis-server-plugins:/srv/plugins \
-	    -v $(shell pwd)/:/src \
-        --entrypoint /src/add_server_plugins.sh \
-	    3liz/qgis-map-server:${LZMQGSRVVERSION} \
+	echo "Plugins are auto installed at qgis worker startup"
 
 show-qgis-server-versions:
 	docker run \
 	    --rm -i \
-	    --entrypoint qgisserver \
-	    3liz/qgis-map-server:${LZMQGSRVVERSION} \
-	    --version
+	    --entrypoint qgis-server-rpc \
+	    3liz/qgis-services:${LZMQGSRVVERSION} \
```

```suggestion
	    3liz/qjazz:${LZMQGSRVVERSION} \
```

QJazz or services ?

Same line L66

---

qjazz

---

So the repo https://hub.docker.com/r/3liz/qgis-services will be removed ?

---

yes